source/slang/slang-emit-c-like.cpp

```diff
@@ -4341,6 +4351,9 @@ void CLikeSourceEmitter::computeEmitActions(IRModule* module, List<EmitAction>&
     ctx.actions = &ioActions;
     ctx.openInsts = InstHashSet(module);
 
+    
+    buildEntryPointReferenceGraph(this->m_referencingEntryPoints, module);
```

Comment:
We should only need this for glsl backend, right?

Reply:
yes

Reply:
I can move the logic to init for GLSL specifically for now.